common/etc/nginx/templates/default.conf.template

```diff
@@ -78,6 +90,7 @@ server {
 
         # Set the Authorization header to the AWS Signatures credentials
         proxy_set_header Authorization $s3auth;
+        proxy_set_header X-Amz-Security-Token $s3SecurityToken;
```

Comment:
We will also need to add this line to the `location @s3Listing` block as well. Without it, directory listings won't work.

Reply:
Good catch! I didn't try out directory listing so far 🙈